.github/actions/doc-deploy/action.yml

```
@@ -0,0 +1,53 @@
+name: Deploy Documentation
+description: Build doc, upload it to S3 and invalidate Cloudfront cache
+
+inputs:
+  AWS_ACCESS_KEY_ID:
+    description: AWS Access key ID
+    required: true
+  AWS_SECRET_ACCESS_KEY:
+    description: AWS secret key
+    required: true
+  S3_BUCKET:
+    description: S3 bucket name
+    required: true
+  CLOUDFRONT_ID:
+    description: Cloudfront distribution ID
+    required: true
+  REGION:
+    description: AWS default region
+    required: true
+  FRAMEWORK_BRANCH:
+    description: Documentation framework branch to use
+    required: true
+  
+runs:
+  using: "composite"
+  steps:
+    - name: Install AWS CLI
+      run: |
+        sudo apt-get update
+        sudo apt-get install python python-pip
+        pip install awscli --upgrade --user
+      shell: bash
+    - name: Build documentation
+      run: |
+        rm -fr doc/framework
+        npm install --production=false
+        npm run doc-prepare
+        npm run doc-build
+      env:
+        NODE_ENV: production
+        BRANCH: ${{ inputs.FRAMEWORK_BRANCH }}
```

```suggestion
        FRAMEWORK_BRANCH: ${{ inputs.FRAMEWORK_BRANCH }}
```
Kuzdoc expects to have the env variable `FRAMEWORK_BRANCH` set